tests/wpt/web-platform-tests/css/cssom/stylesheet-replacedata-dynamic.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<title>(Test #1) Stylesheet replaceData Dynamic</title>
+<link rel="match" href="stylesheet-replacedata-dynamic-ref.html">
+<link rel="author" title="Emilio Cobos Álvarez" href="mailto:ecobos@igalia.com">
```

Comment:
You should add yourself to the list here, if you want (if you don't, that's fine too :P).

Also, please use emilio@crisal.io or emilio@mozilla.com, since I no longer do stuff for Igalia :)